--- a/lib/mcollective/connector/activemq.rb
+++ b/lib/mcollective/connector/activemq.rb
@@ -228,7 +228,7 @@ module MCollective
           1.upto(pools) do |poolnum|
             host = {}
 
-            host[:host] = get_option("activemq.pool.#{poolnum}.host")
+            host[:host] = get_env_or_option("STOMP_HOST", "activemq.pool.#{poolnum}.host")
             host[:port] = get_option("activemq.pool.#{poolnum}.port", 61613).to_i
             host[:login] = get_env_or_option("STOMP_USER", "activemq.pool.#{poolnum}.user", '')
             host[:passcode] = get_env_or_option("STOMP_PASSWORD", "activemq.pool.#{poolnum}.password", '')
